addinstanceform.php

```diff
@@ -39,7 +39,9 @@ class cachestore_redis_addinstance_form extends cachestore_addinstance_form {
     protected function configuration_definition() {
         $form = $this->_form;
 
-        $form->addElement('text', 'server', get_string('server', 'cachestore_redis'), array('size' => 24));
+        $form->addElement('checkbox', 'clustermode', 'Cluster Mode');
```

Comment:
Right. I can test if it is available with `class_exists('RedisCluster')`, in that case I can disable/hide the checkbox and add a static message.